test/node/parsing/UIEvent/5

```diff
@@ -0,0 +1,10 @@
+dictionary MouseEventInit : EventModifierInit {
```

Comment:
Some of the tests use to contain `index` fragments, so I revised just changed the tests to use non-index fragments.